diagnostic.js

```diff
@@ -30,25 +30,63 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const controller = require('lib/wiring/controller');
+const models = require('app/models');
+const Art = models.art;
+
+
+const create = (req,res,next) => {
+  Art.create(req.body.art)
+  .then(art => {})
+  .then(art => res.json({ art }))
+  .catch(err => next(err));
+};
```

Comment:
We are not working within Express right now. Consequently, `req` and `res` do not exist.